src/components/PrivateRoute.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { Redirect, Route, RouteComponentProps, RouteProps } from 'react-router-dom';
+import LoginStore from '~/stores/loginStore';
+
+type RouteComponent = React.FunctionComponent<RouteComponentProps<{}>> | React.ComponentClass<any>;
+
+/* tslint:disable:variable-name */
```

Comment:
Why disable this? I suppose since variables are required to be in camel-case, but a better solution would maybe be to edit the tslint config, or to comply with its rules

Reply:
Added tslint rule